### PR TITLE
Put the unittest documentation into the readme

### PR DESCRIPTION
**Pull request by OlegHahm**
I was already several times annoyed from being pointed to the wiki - particular while being without a Internet connection.

EDIT: this is just an import from https://github.com/RIOT-OS/RIOT/wiki/Testing-RIOT

### CODE REVIEW

**Comment by OlegHahm** [comment]
I would replace the wiki enty with a pointer to the README if we agree on this.

**Comment by LudwigOrtmann** [comment]
I am in favor.

File: tests/unittests/README.md [hunk 1]
```diff
@@ -1,6 +1,325 @@
...
+
+1. **[Create a Makefile](#create-a-makefile)**: add a file ``tests-<modulename>/Makefile``
+2. **[Define a test header](#define-a-test-header)**: add a file ``tests-<modulename>/tests-<modulename>.h``
+3. **[Implement tests](#implement-tests)**: for each header file, that defines a function or macro implemented or related to the module, add a file ``tests-<modulename>/tests-<modulename>-<headername>.c`` or ``tests-<modulename>/tests-<modulename>.c`` if there is only one header.
+
+#### Create a Makefile
+The Makefile should have the following content:
+
+```Makefile
+MODULE = tests-<modulename>
```
Comment (authmillenon) [inline tests/unittests/README.md]: I think this line is not needed anymore

**Comment by authmillenon** [comment]
ACK.

**Pull request merged**